### PR TITLE
feat: Beta support pages

### PR DESCRIPTION
**Pull request by sarahmonster**
This PR adds a roadmap and privacy policy page to the site, in order to better support our beta programme.

I haven't tested these out locally just yet since I need to do a little work to be able to build the site, but once I can do that I'll tweak any visual stuff that may need tweaking and/or make stuff prettier.

Fix https://github.com/octopusthink/turnip-timer/issues/169

### CODE REVIEW

**Comment by sarahmonster** [comment]
@tofumatt I think this is ready for a review whenever you're ready! Privacy policy is based on Mic Drop's but I've added some additional bits about newsletters and such, and I've made a few small tweaks as well to the main site as a whole.

File: src/components/SignupForm/index.js [hunk 1]
```diff
@@ -85,12 +85,12 @@ const SignupForm = () => {
...
-          max-width: 48rem;
+          max-width: 56rem;
+          font-size: 1.6rem;
         `}
       >
-        Your information will only be used to send beta notifications, and will never be sold to
-        anyone. You can unsubscribe at any time. Beta users will receive discounts on the full
-        product and our undying affection.
+        Your information will only be used to send beta notifications.{' '}
+        <Link to="/privacy">Privacy.</Link>
```
Comment (tofumatt) [inline src/components/SignupForm/index.js:93]: This single-word sentence strikes me as odd. What about:

```suggestion
        <Link to="/privacy"><VisuallyHidden>Read out </VisuallyHidden>Privacy<VisuallyHidden> Policy</VisuallyHidden></Link>
```

File: src/content/pages/privacy.mdx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+# Privacy policy
+
+Turnip is a simple timer app made by Sarah &amp; Matt at [Octopus Think](https://octopusthink.com/about).
+
+By default, Turnip collects **absolutely no data** about you, your computer, or your usage of the app.
+
+## Beta programme
+
+If you sign up for the Turnip beta, crash reporting and analytics are mandatory. This is to ensure that we know if anything is going wrong. Your data is anonymised and isn’t tied to any personal identifier in any way. If that makes you uncomfortable, we totally understand, but we think this is the best way to ensure that we can improve the app for our beta users as quickly and efficiently as possible.
```
Comment (tofumatt) [inline src/content/pages/privacy.mdx:20]: So: in the future, we will likely end up associating a crash report with a user ID. This allows us to check out non-private user data associated with an account for crashes.

Right now there aren't any identifiers, but just something to think about...

File: src/content/pages/privacy.mdx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+## Integrations
+
+Turnip connects to your third-party services using secure authentication tokens provided by the third-party service. We don't store, collect, or even see your password or login credentials for any third-party service.
```
Comment (tofumatt) [inline src/content/pages/privacy.mdx:24]: This phrasing is _technically_ true, but it sort of makes it sound like we don't store those obtained authentication tokens. It's also quite technical and not really about privacy, more about security.

For what it's worth, I suspect we _will_ store OAuth tokens for things like FreeAgent in the future so we can sync them for users and save them the hassle of having to sign in again on a new Mac, etc.

```suggestion
Turnip connects to your third-party services using secure authentication tokens provided by the third-party service. We don't store, collect, or even see your password or login credentials for any third-party service. 
```

File: src/content/pages/privacy.mdx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+**Analytics are completely optional and opt-in.** We'll never sell your information to advertisers, third-parties, or that kid who was mean to you in first grade. If you opt into analytics, we'll always ensure that your data is anonymised and kept secure.
+
+If you do choose to opt into our analytics, here's what we collect:
+- Anonymous usage statistics—stuff like what integrations you use, how many tags you use for a timeslip, and the number of times you interact with the app.
+- Your country and OS language.
+- The version number of Turnip you're using.
+- Crash logs and info about audio devices connected to your Mac.
+- The type of Mac you have and the version of macOS you're running.
+
+If you opt-in, this usage data is uploaded to our Firebase Analytics account, but contains no personally identifiable information about you or your usage of Turnip.
```
Comment (tofumatt) [inline src/content/pages/privacy.mdx:37]:
```suggestion
If you opt-in, usage data is uploaded to our Firebase Analytics account and crash reports are uploaded to our Sentry account. This data contains no personally identifiable information about you or your usage of Turnip.
```

**Review by tofumatt — COMMENTED** [review]
Looks great! A few comments and suggestions, but overall looks good 👍🏻

File: src/content/pages/privacy.mdx (hunk 3 above)
Comment (sarahmonster) [reply src/content/pages/privacy.mdx:24]: I think I'm mostly trying to say "we don't save your password" here, and clarify that we don't collect data associated with their <whatever> account. 

Is this the suggested change you meant to make? I've compared them like six times and I really can't find a difference! 😅

**Pull request merged**